### PR TITLE
Deprecation contains -> occursin

### PR DESCRIPTION
**Pull request by mauro3**

### CODE REVIEW

File: src/utils.jl [hunk 1]
```diff
@@ -128,7 +128,7 @@ function inexpr(ex, x)
   return result
 end
 
-isgensym(s::Symbol) = contains(string(s), "#")
+isgensym(s::Symbol) = contains("#", string(s))
```
Comment (MikeInnes) [inline src/utils.jl]: Is this meant to be `occursin`?

Comment (mauro3) [reply src/utils.jl]: Oops, good job you check!

**Comment by mauro3** [comment]
Bump

**Comment by MikeInnes** [comment]
:+1: Thanks!

**Pull request merged**